liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseHistoryMojo.java

```diff
@@ -14,10 +16,16 @@ public class LiquibaseHistoryMojo extends AbstractLiquibaseMojo {
 
     @Override
     protected void performLiquibaseTask(Liquibase liquibase) throws LiquibaseException {
-      CommandScope historyCommand = new CommandScope(InternalHistoryCommandStep.COMMAND_NAME);
-
-      historyCommand.addArgumentValue(InternalHistoryCommandStep.DATABASE_ARG, getLiquibase().getDatabase());
-
-      historyCommand.execute();
+        ClassLoader mavenClassLoader;
```

Comment:
Same as above - could this ClassLoader getter be moved to `getDatabase/createDatabase` ? And here we have a try/catch, above we don't.